### ISSUE TITLE
ffho-autoupdater-wifi-fallback: Add instructions, how to install 

### ISSUE THREAD

**Issue by rubo77** [FreifunkHochstift/ffho-packages#5]
Please add detailed instructions with a working example, how to add the package to build a firmware in https://github.com/FreifunkHochstift/ffho-packages/tree/master/ffho/ffho-autoupdater-wifi-fallback

**Comment by kb-light** [comment]
that should be something like:
```
cd gluon
git checkout v2016.2.x
git checkout -b v2016.2.x-patched
wget https://git.c3pb.de/freifunk-pb/site-ffho/raw/stable/patches/0001-openwrt-patch-iw.patch
git am 0001-openwrt-patch-iw.patch
```
and adding the repository to `modules` and the package to the `site.mk`.

Please keep in mind, that lede is currently not supported by our packages.

**Comment by rubo77** [comment]
And gluon-luci-autoupdater durch ffho-luci-autoupdater-wifi-fallback ersetzen.

**Comment by rubo77** [comment]
And the wget needs a 

    --no-check-certificate

so it will be 

    wget --no-check-certificate https://git.c3pb.de/freifunk-pb/site-ffho/raw/stable/patches/0001-openwrt-patch-iw.patch

**Comment by kb-light** [comment]
Yes, there is currently an issue with the ssl-cert. (we are going to move to our own git soon)

**Comment by rubo77** [comment]
where does the patch have to go? in which folder? I cannot build this, please add a complete instruction soon ;)

**Comment by kb-light** [comment]
the patch should be applied to your desired gluon branch/commit. After applying it you should see an additional patch in `./patches/openwrt`.

**Comment by rubo77** [comment]
i thhink I have a complete german install instruction together now here: https://forum.freifunk.net/t/fork-von-ffho-autoupdater-wifi-fallback-gesucht/11423/31
